draft-cutler-httpbis-partitioned-cookies.md

```diff
@@ -85,8 +88,9 @@ This algorithm could be added after {{Section 5.2 ("Same-site" and "cross-site"
 
 {:quote}
 > 1. Let top-document be the active document in document's browsing context's top-level browsing context.
-> 2. Let "cookie-partition-key" be the site of the top-document when the user agent made the request.
-> 3. If the cookie is being read or written via a "non-HTTP" API, then cookie-partition-key is the site (as defined in {{HTML}}) of the top-document of the document associated with the non-HTTP API.
+> 2. Let "cookie-partition-key" be a tuple whose first element is the site of the top-document when the user agent made the request and whose second element is a boolean (cross-site ancestor) indicating if the cookie was created in a cross-site context as defined in [Section 5.2 of RFC6265bis](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis-13#section-5.2).
```

Comment:
Let's use an appositive here instead, and let's also keep it all one word in kebab-case.

`whose second element is a boolean, which we will call "cross-site-ancestor",`